idaes/generic_models/unit_models/plate_heat_exchanger.py

```diff
@@ -179,86 +182,89 @@ def build(self):
-                "The number of elements in the channel list: {}  "
-                " does not match the number of passes ({}) given. "
-                "Please provide as integers, the number of channels of each pass".format(
-                    self.config.channel_list, self.config.passes))
+                f"The number of elements in the channel list ("
+                f"{self.config.channel_list}) does not match the number of "
+                f"passes ({self.config.passes}) given. Please provide as "
+                f"integers, the number of channels of each pass")
 
         # ======================================================================
         # Build hot-side  Control Volume (Lean Solvent)
-        self.hot_side = ControlVolume0DBlock(default={
+        self.hot_fluid = ControlVolume0DBlock(default={
             "dynamic": self.config.dynamic,
             "has_holdup": self.config.has_holdup,
             "property_package": self.config.hot_side.property_package,
-            "property_package_args": self.config.hot_side.property_package_args})
+            "property_package_args":
+                self.config.hot_side.property_package_args})
 
-        self.hot_side.add_state_blocks(has_phase_equilibrium=False)
+        self.hot_fluid.add_state_blocks(has_phase_equilibrium=False)
 
-        self.hot_side.add_material_balances(
+        self.hot_fluid.add_material_balances(
             balance_type=MaterialBalanceType.componentTotal,
             has_mass_transfer=False,
             has_phase_equilibrium=False,
             has_rate_reactions=False)
 
-        self.hot_side.add_momentum_balances(
+        self.hot_fluid.add_momentum_balances(
             balance_type=MomentumBalanceType.pressureTotal,
             has_pressure_change=True)
 
         # Energy balance is based on the effectiveness Number of Transfer units
-        # (E-NTU method) and inluded as performance equations. Hence the control
-        #  volume energy balances are not added.
+        # (E-NTU method) and inluded as performance equations. Hence the
+        # control volume energy balances are not added.
 
-        # ======================================================================
+        # =====================================================================
         # Build cold-side  Control Volume(Rich solvent)
-        self.cold_side = ControlVolume0DBlock(default={
+        self.cold_fluid = ControlVolume0DBlock(default={
             "dynamic": self.config.dynamic,
             "has_holdup": self.config.has_holdup,
             "property_package": self.config.cold_side.property_package,
-            "property_package_args": self.config.cold_side.property_package_args})
+            "property_package_args":
+                self.config.cold_side.property_package_args})
 
-        self.cold_side.add_state_blocks(has_phase_equilibrium=False)
+        self.cold_fluid.add_state_blocks(has_phase_equilibrium=False)
 
```

Comment:
Where is the energy balance for the system? Even with the NTU calculations, I would expect to see an proper energy balance for each control volume with a heat transfer term, a constraint that says Qhot = -Qcold, and then the NTU calculations used to determine the value of one of the Q terms.